src/bindings/python/tests/test_inference_engine/test_function.py

```diff
@@ -349,3 +349,15 @@ def test_reshape_with_names():
     for input in model.inputs:
         model.reshape({input.any_name: new_shape})
         assert input.partial_shape == new_shape
+
+
+def test_reshape(device):
+    shape = Shape([1, 10])
+    param = ops.parameter(shape, dtype=np.float32)
+    model = Model(ops.relu(param), [param])
+    ref_shape = model.input().partial_shape
+    ref_shape[0] = Dimension(3)
```

Comment:
```suggestion
    ref_shape[0] = 3
```
Is it possible?

Reply:
No, but we can make it possible in python, adding one more overload of `__setitem__`

Reply:
Please add it. I see it for `Shape` but there is no such overload for `PartialShape`...

Reply:
Done.